plugins/services/src/js/components/__tests__/__snapshots__/ServiceList-test.js.snap

```diff
@@ -16,11 +16,16 @@ exports[`ServiceList #getServices returns services that have a value of two elem
       </span>
```

Comment:
this is the updated snapshot

Reply:
![social-lg](https://user-images.githubusercontent.com/1527504/33744694-5bc3cc10-db67-11e7-93ce-77c550cc51f4.jpg)